ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -565,21 +529,21 @@ class Tab: NSObject {
     // To remove history from WebKit, we simply update the session data AFTER calling "clear" above
     SessionTab.update(
       tabId: id,
-      interactionState: webView.sessionData ?? Data(),
+      interactionState: webView.underlyingWebView?.sessionData ?? Data(),
```

Comment:
need to replace restoration with Chromium WebState restoration otherwise WebState can be out of sync with WKWebView and hit CHECKs